### DISCUSSION TITLE
My prediction intervals are way too small

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2422] [Q&A]
Is there any way that the forecast intervals could be larger?
My NRMSE is actually already pretty good, but the forecast intervals are so narrow that they do not cover any values at all.
Does anyone have an idea how to improve this?

**Comment by jaheba** [comment]
This sounds like you are overfitting the model. One solution for this is to use a validation dataset to prevent overfitting.

**Reply by MarleneBs** [reply]
Thank you very much, can you tell me how to do this? I am not sure. I used my data in the way that is discribed in pandas.Dataframe based datasets and i also use feat.
